packages/common/src/logger/logger.ts

```diff
@@ -1,22 +1,25 @@
 // Copyright 2020-2021 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import fs from 'fs';
 import {stringify} from 'flatted';
 import Pino, {LevelWithSilent} from 'pino';
 import {colorizeLevel, ctx} from './colors';
 
+const rotator = require('file-stream-rotator');
```

Comment:
use import

Reply:
changes to a different package with better typescript support.